### PR TITLE
system-veriables: add @@tidb_min_paging_size and @@tidb_max_paging_size

### PR DESCRIPTION
**Pull request by tiancaiamao**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/pull/36107

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1339,6 +1339,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 范围：`[32, 2147483647]`
 - 这个变量用来设置执行过程中一个 chunk 最大的行数，设置过大可能引起缓存局部性的问题。
 
+### `tidb_max_paging_size`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`50000`
+- 范围：`[1, 2147483647]`
+- 这个变量用来设置 coprocessor 协议中 paging size 的最大的行数，设置过小会造成与 TiKV 的 RPC 交互次数增加，设置过大则会造成导数据/全表扫等特定场景的内存使用变高。
```
Comment (shichun-0415) [inline system-variables.md:1348]:
```suggestion
- 这个变量用来设置 coprocessor 协议中 paging size 的最大的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，导数据和全表扫等特定场景会占用更多内存。
```

File: system-variables.md [hunk 2]
```diff
@@ -1428,6 +1436,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 单位：秒
 - 这个变量设置了查询 `METRIC_SCHEMA` 时生成的 Prometheus 语句的 step。
 
+### `tidb_min_paging_size`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`128`
+- 范围：`[1, 2147483647]`
+- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数，设置过小会造成与 TiKV 的 RPC 交互次数增加。设置过大可能造成 IndexLookup 带 Limit 场景下的性能下降。
```
Comment (shichun-0415) [inline system-variables.md:1445]: 设置过小，应该是交互减少？
```suggestion
- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB与 TiKV 的 RPC 交互会减少；设置过大，IndexLookup 带 Limit 场景会出现性能下降。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -1339,6 +1339,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 范围：`[32, 2147483647]`
 - 这个变量用来设置执行过程中一个 chunk 最大的行数，设置过大可能引起缓存局部性的问题。
 
+### `tidb_max_paging_size`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`50000`
+- 范围：`[1, 2147483647]`
```
Comment (shichun-0415) [inline system-variables.md:1347]:
```suggestion
- 范围：`[1, 2147483647]`
- 单位：行
```

File: system-variables.md [hunk 4]
```diff
@@ -1428,6 +1436,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 单位：秒
 - 这个变量设置了查询 `METRIC_SCHEMA` 时生成的 Prometheus 语句的 step。
 
+### `tidb_min_paging_size`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`128`
+- 范围：`[1, 2147483647]`
```
Comment (shichun-0415) [inline system-variables.md:1444]:
```suggestion
- 范围：`[1, 2147483647]`
- 单位：行
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by lcwangchao — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -1339,6 +1339,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 范围：`[32, 2147483647]`
 - 这个变量用来设置执行过程中一个 chunk 最大的行数，设置过大可能引起缓存局部性的问题。
 
+### `tidb_max_paging_size`
```
Comment (shichun-0415) [inline system-variables.md:1342]:
```suggestion
### `tidb_max_paging_size` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: system-variables.md [hunk 6]
```diff
@@ -1428,6 +1437,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 单位：秒
 - 这个变量设置了查询 `METRIC_SCHEMA` 时生成的 Prometheus 语句的 step。
 
+### `tidb_min_paging_size`
```
Comment (shichun-0415) [inline system-variables.md:1440]:
```suggestion
### `tidb_min_paging_size` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: system-variables.md [hunk 7]
```diff
@@ -1339,6 +1339,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 这个变量用来设置执行过程中一个 chunk 最大的行数，设置过大可能引起缓存局部性的问题。
 
+### `tidb_max_paging_size`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`50000`
+- 范围：`[1, 2147483647]`
+- 单位：行
+- 这个变量用来设置 coprocessor 协议中 paging size 的最大的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，导数据和全表扫等特定场景会占用更多内存。
```
Comment (TomShawn) [inline system-variables.md:1349]: 能否给“合理”一个参考呢？用户如何得知自己设的值过大还是过小呢？

Comment (tiancaiamao) [reply system-variables.md:1349]: 没法给"合理"的值，对于 TP 场景，我认为默认值就是很合理的。
但是放到 AP 场景，可能又不算特别合理了。
用户(包括我们自己)不去做很多 benchmark，是没法知道多少是合理的，就只好不建议用户瞎调了。

Comment (TomShawn) [reply system-variables.md:1349]: 那能否说一下不建议调该变量呢？

**Comment by easonn7** [comment]
LGTM

**Pull request merged**